tests/checkers/unittest_utils.py

```diff
@@ -163,9 +163,9 @@ def test_parse_format_method_string() -> None:
         ("{!r:}", 1),
     ]
     for fmt, count in samples:
-        keys, num_args, pos_args = utils.parse_format_method_string(fmt)
+        keys, num_args, pos_args, _, _, _ = utils.parse_format_method_string(fmt)
         keyword_args = len({k for k, l in keys if not isinstance(k, int)})
-        assert keyword_args + num_args + pos_args == count
+        assert keyword_args + num_args + len(pos_args) == count
```

Comment:
In addition to the requested functional tests from Pierre I think it would be best to also include unit tests that check if the new return values work as expected. I.e. if `keyword_types`, `implicit_types` and `explicit_types` work as intended.